bulk-model-sync-gradle/src/main/kotlin/org/modelix/model/sync/bulk/gradle/config/ModelSyncGradleSettings.kt

```diff
@@ -68,6 +69,10 @@ data class SyncDirection(
         target = endpoint
     }
 
+    fun mpsModulesFolder(folder: File) {
```

Comment:
Is the different in spelling between "moduleFolders" and "module_s_Folder" intentional? Slightly confusing for users.

Reply:
Renamed and documented it.